# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../vdr-mplayer/mp3-ctype-include.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- mp3/mplayer.c.orig	2006-09-19 18:18:13.000000000 +0000
+++ mp3/mplayer.c	2006-11-02 13:18:32.000000000 +0000
@@ -22,6 +22,7 @@
 #include <getopt.h>
 #include <malloc.h>
 #include <stdlib.h>
+#include <ctype.h>
 
 #include "common.h"
 
